benchmarks/benchmark.py

```diff
@@ -74,11 +82,14 @@ def benchmark(golden_rules, tokenize_func):
         syntok_tokenize)
     for tokenize_func in libraries:
         t = time.time()
-        for i in range(100):
-            percent_score = benchmark(GOLDEN_EN_RULES, tokenize_func)
+        # wget http://www.gutenberg.org/files/1661/1661-0.txt -P benchmarks/
```

Comment:
I'd prefer if you didn't comment this out, and instead made a new benchmark script for the longer doc.

Reply:
Moved to new script